keep/api/core/db_on_start.py

```diff
@@ -42,7 +42,7 @@
 engine = create_db_engine()
 
 
-def try_create_single_tenant(tenant_id: str) -> None:
+def try_create_single_tenant(tenant_id: str, create_default_user=True) -> None:
```

Comment:
why is this needed?

Reply:
cuz with oauth2proxy we don't want default user